app/web/src/components/Navigation/Navigation.tsx

```diff
@@ -241,17 +393,47 @@ export default function Navigation() {
             </div>
           </Hidden>
         </div>
-        <div className={classes.bug}>
-          <Hidden smDown>
-            <ExternalNavButton
-              route={handbookURL}
-              label={HELP}
-              labelVariant="h3"
-            />
-            <NavButton route={logoutRoute} label={LOG_OUT} />
-          </Hidden>
-          <ReportButton />
-        </div>
+
+        <Hidden smDown>
+          <div ref={menuRef} style={{ display: "flex", flexDirection: "row" }}>
+            <div
+              style={{ marginRight: 16, display: "flex", alignItems: "center" }}
```

Comment:
Also, we don't need this extra `div` - instead, you can make the outer `div` on line 398 to have `alignItems: center` and apply the margin on the button instead.

Another thing - use the spacing in our theme - e.g. `theme.spacing(2)` when you move this to the `useStyles` instead of a literal pixel value, again for more consistency.